README.md

```diff
@@ -50,12 +54,12 @@ The choices are `border`, `outline`, `padding`, and `margin`. You will leave two
 Your Answer:
 
 ```text
-___: Defines the distance between an element's border and adjacent elements' borders.
-___: Inserts a "wall" around an element.
-___: Defines the distance between an element's content and its border.
-___: Defines the width of an element.
-___: Overlays a "wall" on top of an element.
-___: Defines the distance between the center of an element and the center of the adjacent element.
+Margin: Defines the distance between an element's border and adjacent elements' borders.
+Border: Inserts a "wall" around an element.
+Padding: Defines the distance between an element's content and its border.
+__: Defines the width of an element.
+Outline: Overlays a "wall" on top of an element. (I found this sentence confusing. Top seems to denote only the top, rather than "over the element as a whole")
+__: Defines the distance between the center of an element and the center of the adjacent element.
 ```
```

Comment:
:+1:

I find the beginning of this blog post to be helpful in describing the difference between `border` and `outline`: https://css-tricks.com/almanac/properties/o/outline/

I wouldn't worry too much about the difference. I don't find myself using `outline` all that much. As the blog post suggests, it's more useful in debugging than it is actual styling.